at_commons/lib/src/keystore/at_key_builder_impl.dart

```diff
@@ -131,3 +131,11 @@ class PrivateKeyBuilder extends AbstractKeyBuilder {
     _meta.isPublic = false;
   }
 }
+
+/// Builder to build the local keys
+class LocalKeyBuilder extends AbstractKeyBuilder {
```

Comment:
Please add unit tests to assert behaviour of LocalKeyBuilder

Reply:
Added test for LocalKeyBuilder